apps/packtool/packtool.cpp

```diff
@@ -326,10 +328,39 @@ int WriteRaw(const std::filesystem::path& outFilename, const std::vector<std::fi
 	return EXIT_SUCCESS;
 }
 
-int WriteMeshFile(const std::filesystem::path& outFilename) noexcept
+int WriteMeshFile(const std::filesystem::path& outFilename, const std::vector<std::filesystem::path> inFilenames) noexcept
```

Comment:
not a const reference of the vector instead?

Reply:
Good catch